tiny-firmware/Makefile

```diff
@@ -36,6 +36,8 @@ OBJS += $(FIRMWARE_SRCS:.c=.o)
 
 OBJS += protob/c/messages.pb.o
 OBJS += protob/c/types.pb.o
+OBJS += protob/c/bitcoin_messages.pb.o
+OBJS += protob/c/bitcoin_types.pb.o
```

Comment:
Rewrite here makefile, so it will include all files from `protob/c`, so nextly we don't need to change Makefile, if add new files in protobuf.